mne_bids/tests/test_mne_bids.py

```diff
@@ -129,5 +132,7 @@ def test_brainvision():
                 output_path=output_path, kind='eeg', eeg_reference='Cz',
                 event_id={'test': 1})
 
-    cmd = ['bids-validator', output_path]
-    run_subprocess(cmd, shell=shell)
+    # Until EEG validator is merged, assert that this will be wrong.
+    with pytest.raises(subprocess.CalledProcessError):
+        cmd = ['bids-validator', output_path]
+        run_subprocess(cmd, shell=shell)
```

Comment:
once the bids-validator can manage EEG, we just habe to remove the `with pytest.raises()` line to make the test work.